packages/eslint-plugin/docs/rules/consistent-type-exports.md

```diff
@@ -70,15 +83,14 @@ export type { ButtonProps } from 'some-library';
 ### ✅ Correct
 
 ```ts
-export { Button } from 'some-library';
-export type { ButtonProps } from 'some-library';
 export { Button, type ButtonProps } from 'some-library';
 ```
 
 ## When Not To Use It
 
 - If you are using a TypeScript version less than 3.8, then you will not be able to use this rule as type exports are not supported.
 - If you specifically want to use both export kinds for stylistic reasons, you can disable this rule.
+- If you use --isolatedModules, you don't need this rule since TS would enforce type-export specifiers.
```

Comment:
Not critical, but I think it's clearer this way:

```suggestion
- If you use `--isolatedModules`, you don't need this rule since the compiler would error if a type is not re-exported using `export type`.
```